tests/test_enhanced_operators_real_api.py

```
@@ -0,0 +1,264 @@
+    try:
+        result1 = await operator.mutate_single(regular_idea, evaluation_context)
+        
+        print(f"✅ Original: {regular_idea.content[:80]}...")
+        print(f"✅ Mutated:  {result1.content[:80]}...")
+        print(f"✅ Operator: {result1.metadata.get('operator')}")
+        print(f"✅ Mutation Type: {result1.metadata.get('mutation_type')}")
+        print(f"✅ Is Breakthrough: {result1.metadata.get('is_breakthrough')}")
+        print(f"✅ Cost: ${result1.metadata.get('llm_cost', 0):.4f}")
+        
+        # Verify it's not a breakthrough mutation
+        assert result1.metadata.get('is_breakthrough') == False
```

Ruff (0.14.5)

Avoid equality comparisons to `False`; use `not result1.metadata.get('is_breakthrough'):` for false checks

Replace with `not result1.metadata.get('is_breakthrough')`

(E712)